crates/scarb-api/src/output_transform.rs

```diff
@@ -0,0 +1,35 @@
+use std::borrow::Cow;
+
+pub trait OutputTransform: Clone + Default {
```

Comment:
I really like the idea of having two different stream transformations available!
Though I think it would be much simpler instead of encoding those as generics to just provide
methods/structs to transform the output. I don't see any benefit of having output transformation being encoded as a part of command definition type.